### PR TITLE
try to fix ipynb

### PR DESCRIPTION
**Pull request by jessica-mitchell**

### CODE REVIEW

File: doc/conf.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
 
 import sys
 import os
-# import shlex
+import shlex
```
Comment (steffengraber) [inline doc/conf.py]: Is there a change?

File: doc/requirements.txt [hunk 2]
```diff
@@ -1,2 +1,5 @@
 sphinx>=1.4
-recommonmark==0.4.0
\ No newline at end of file
+recommonmark==0.4.0
```
Comment (steffengraber) [inline doc/requirements.txt:2]: Is there a change?

**Comment by steffengraber** [comment]
Is it not better to make changes to 'IAF_neurons_singularity.ipynb' as an issue in nest/nest-simulator?

File: doc/conf.py (hunk 1 above)
Comment (jessica-mitchell) [reply doc/conf.py]: no but I dont think it should be commented-out

File: doc/requirements.txt (hunk 2 above)
Comment (jessica-mitchell) [reply doc/requirements.txt:2]: er no, this line is the same

File: doc/conf.py (hunk 1 above)
Comment (steffengraber) [reply doc/conf.py]: shlex isn't used. Or is a needed in an other way?

**Comment by steffengraber** [comment]
Are there any news?

**Comment by steffengraber** [comment]
@jessica-mitchell Are there any news?

**Comment by jessica-mitchell** [comment]
I think this PR can be closed because, since then, worked on testing this and other changes may be needed

**Pull request closed without merging**